.github/workflows/protectSecurityRelevantCode.yml

```
@@ -0,0 +1,172 @@
+            ##### set action output to true
+            echo -e "\033[31mThe following security-relevant files were are changed by this PR:\033[0m"
+            echo "$PROTECTED_FILES"
+            echo "CONTINUE=true" >> $GITHUB_ENV
+          fi
+
+      - name: Get "Information Security Manager" Group Members
+        if: env.CONTINUE == 'true'
+        env:
+          GH_PAT: ${{ secrets.GIT_ACTIONS_BOT_PAT_CLASSIC }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:6:6: Double quote to prevent globbing and word splitting

(shellcheck)